### DISCUSSION TITLE
为什么不早run呢？

### DISCUSSION THREAD

**Discussion by WWz33** [The-Run-Philosophy-Organization/run#400] [General]
为什么2019年没有疫苗、没变异没给你免费医疗的之前不快run呢？现在毒效变弱了才想着润啊，老墙头草了

**Comment by the-run-philosophy** [comment]
不早RUN？？？
没有我们早RUN的人谁给你们做GITHUB？等着在墙内被抓走枪毙？？

**Comment by wedxz** [comment]
有多少人是因为国内的疫情防控政策才下定决定要run的,这政策伤了多少人的心
